src/features/transfer-ownership/TransferOwnership.utils.ts

```diff
@@ -0,0 +1,15 @@
+import { isValidEthAddress } from '../../lib/util/address/address';
+
+export const isValid = (value: string, account: string) =>
+	isValidEthAddress(value) && !(value.toLowerCase() === account.toLowerCase());
+
+export const handleInputError = (
+	value: string,
+	owner: string,
+	account: string,
+) =>
+	account.toLowerCase() !== owner.toLowerCase()
+		? 'You are not the owner'
+		: isValidEthAddress(value) && value.toLowerCase() === account.toLowerCase()
+		? 'The address entered already owns this domain'
+		: 'Please enter a valid Ethereum wallet address';
```

Comment:
nit: potentially wrap this in curly brackets - the nested ternary makes this one look a bit daunting.

Reply:
I think there is a minor bug for this. Going to take another look and maybe change to use if/else